core/templates/services/date-time-format.service.spec.ts

```diff
@@ -18,6 +18,18 @@
 
 import { DateTimeFormatService } from 'services/date-time-format.service';
 
+// Needed to avoid typescript errors.
```

Comment:
Elaborate this comment more (i.e. explain the exact issue).

Reply:
Done!